test/test_dataloader.py

```diff
@@ -49,7 +49,7 @@
     mp = mp.get_context(method='spawn')
 
 
-JOIN_TIMEOUT = 17.0 if (IS_WINDOWS or IS_PPC) else 11.0
+JOIN_TIMEOUT = 17.0 if (IS_WINDOWS or IS_PPC) else 13.0
```

Comment:
Magic numbers in the air :o)

Reply:
haha yeah, but I doubt the problem is insufficient timeout. I suspect that there is some random system error happening. The improve error message should tell us more if it happens again.